app/assets/stylesheets/components/common/_skip_to_content.scss

```diff
@@ -11,8 +11,8 @@
   top: -1000px;
   left: 0;
   background-color: $color-skip-to-content;
-  z-index: 12;
-  
+  z-index: 210;
```

Comment:
Since this is an accessibility ticket it would be good to get this z-index under 100 as we know that high z-index causes issues with the Dragon screen reader. I think this would mean changing the `.l-header__content` z-index as well. Not sure how much of a can of worms this would be opening up?